source/numerics.tex

```diff
@@ -1786,6 +1786,30 @@
 \indextext{requirements!uniform random bit generator|)}%
 \indextext{uniform random bit generator!requirements|)}
 
+\pnum
+The \tcode{UniformRandomBitGenerator} concept is a slight relaxation of the uniform random bit generator requirements, in that it does not require a nested \grammarterm{typedef-name} \tcode{result_type}.
```

Comment:
Please add line breaks after at most 79 chars, preferably "semantic line breaks", i.e. do not break in the middle of a phrase or term.